rules-reviewed/hibernate/hibernate-catchall.windup.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="hibernate-catchall" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
I'm -1 on switching these to just "Mandatory". However, might it be possible to use a new issue category for these?

Perhaps "Mandatory for Hibernate Upgrade" or something along those lines (preferably something less verbose)? It is certainly a valid point that someone could migrate an application without changing these.